sentry-types/src/protocol/envelope.rs

```diff
@@ -256,13 +441,19 @@ impl From<Transaction<'static>> for Envelope {
 
 #[cfg(test)]
 mod test {
-    use std::time::{Duration, SystemTime};
+    use std::{
+        str::FromStr,
+        time::{Duration, SystemTime},
+    };
 
     use time::format_description::well_known::Rfc3339;
     use time::OffsetDateTime;
 
     use super::*;
-    use crate::protocol::v7::{SessionAttributes, SessionStatus, Span};
+    use crate::protocol::{
+        latest::Level,
+        v7::{SessionAttributes, SessionStatus, Span},
```

Comment:
`latest == v7`, so I guess this was an auto-import. Might be worth deduplicating.